inc/EventDrivenPrognoserBuilder.h

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2018 United States Government as represented by the
+// Administrator of the National Aeronautics and Space Administration.
+// All Rights Reserved.
+
+#ifndef PCOE_EVENTDRIVENPROGNOSERBUILDER_H
+#define PCOE_EVENTDRIVENPROGNOSERBUILDER_H
+
+#include <string>
+#include <vector>
+
+#include "ConfigMap.h"
+#include "EventDrivenPrognoser.h"
+
+namespace PCOE {
+    const extern std::string LOAD_ESTIMATOR_KEY;
```

Comment:
No need to define these in the header. I would argue that the actual variables are implementation details. If we want to define something in the header, we should document the actual values of these strings in the doc-comments for the class.

Reply:
I disagree. Defining them here allows you to programmatically assign them in ways independent of the actual string. 

For example: builder.set(LOAD_ESTIMATOR_KEY, "example");

Reply:
Hmmm. I can see how that would be useful in some cases, but for that use case to work I think the values need to be much more narrowly defined to prevent collisions. I think making them static class members instead of global constants would work for me.